Exercise02/Program.cs

```diff
@@ -0,0 +1,84 @@
+﻿using Exercise02.Model;
+using LINQPad;
+using LINQPad.FSharpExtensions;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Exercise02
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            var data = new CreateData();
+
+            var listOfStudent = data.GetStudents();
+            listOfStudent.ForEach(Console.WriteLine);
+            Console.WriteLine();
+
+            var listOfClass = data.GetClasses();
+            listOfClass.ForEach(Console.WriteLine);
+            Console.WriteLine();
+
+            var list = listOfStudent.OrderBy(x=>x.ClassId).ThenBy(x=>x.Name).ToList();
+            list.ForEach(Console.WriteLine);
+            Console.WriteLine();
+
+            var listHasStudentCount = listOfClass.GroupJoin(

```

Comment:
Try to use [linq syntax](https://www.dotnetperls.com/join)

Reply:
Done